modules/aws-installing-an-aws-load-balancer-operator.adoc

```diff
@@ -240,128 +246,112 @@ EOF
-
++
 * Private subnets: `kubernetes.io/role/internal-elb:1`
 * Public subnets: `kubernetes.io/role/elb:1`
 +
 [NOTE]
 ====
-Internet-facing and internal load balancers will be created within the AZ to which these subnets belong.
+Internet-facing and internal load balancers will be created within the AWS Availability Zone to which these subnets belong.
 ====
 +
-For more information on adding tags to AWS resources, including VPCs and subnets, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tag your Amazon EC2 resources].
-+
 [IMPORTANT]
 ====
-ELBv2 resources (such as ALBs and NLBs) created by ALBO do not inherit custom tags set for
-ifndef::openshift-rosa[]
-{product-title}
-endif::openshift-rosa[]
-ifdef::openshift-rosa[]
-ROSA
-endif::openshift-rosa[]
-clusters. You must set tags separately for these resources.
+ELBv2 resources (such as ALBs and NLBs) created by AWS Load Balancer Operator do not inherit custom tags set for ROSA clusters. You must set tags separately for these resources.
 ====
+
+. Create the AWS Load Balancer Operator by completing the following steps:
 +
-. Create ALBO:
+.. Create an `OperatorGroup` object by running the following command:
 +
-[source,yaml]
+[source,terminal]
 ----
+$ cat EOF | oc apply -f -
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
   name: aws-load-balancer-operator
   namespace: aws-load-balancer-operator
 spec:
-  upgradeStrategy: Default
----
+  targetNamespaces: []
+EOF
+----
++
+.. Create a `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ cat EOF | oc apply -f -
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
   name: aws-load-balancer-operator
   namespace: aws-load-balancer-operator
 spec:
-  channel: stable-v1.0
-  installPlanApproval: Automatic
+  channel: stable-v1
   name: aws-load-balancer-operator
   source: redhat-operators
   sourceNamespace: openshift-marketplace
-  startingCSV: aws-load-balancer-operator.v1.0.0
+  config:
+    env:
+    - name: ROLEARN
+      value: "<operator_role_arn>" <1>
+EOF
 ----
-+
-. Create an AWS ALBC:
+<1> Specifies the ARN role for the {aws-short} Load Balancer Operator. The `CredentialsRequest` object uses this ARN role to provision the {aws-short} credentials. An example of `<operator_role_arn>` is `arn:aws:iam::<aws_account_number>:role/albo-operator`.
+
+. Create an `AWSLoadBalancerController` object, so that you can create an AWS Load Balancer Controller:
 +
 [source,yaml]
 ----
-apiVersion: networking.olm.openshift.io/v1
+$ cat EOF | oc apply -f -
+apiVersion: networking.olm.openshift.io/v1 <1>
 kind: AWSLoadBalancerController
 metadata:
-  name: cluster
+  name: cluster <2>
 spec:
-  subnetTagging: Manual
-  credentials:
-    name: aws-load-balancer-controller-cluster
+  credentialsRequestConfig:
+    stsIAMRoleARN: <albc_role_arn> <3>
```

Comment:
Similar to the operator role's comment: we don't need to create a secret for the controller since the controller role is passed in the custom resource now.

Reply:
Removed: 

```
. Create an `AWSLoadBalancerController` object, so that you can create an AWS Load Balancer Controller:
+
[source,yaml]
----
$ cat EOF | oc apply -f -
apiVersion: networking.olm.openshift.io/v1 <1>
kind: AWSLoadBalancerController
metadata:
  name: cluster <2>
spec:
  credentialsRequestConfig:
    stsIAMRoleARN: <albc_role_arn> <3>
EOF
----
<1> Defines the `AWSLoadBalancerController` object.
<2> Defines the AWS Load Balancer Controller name. All related resources use this instance name as a suffix.
<3> 	Specifies the ARN role for the AWS Load Balancer Controller. The `CredentialsRequest` object uses this ARN role to provision the AWS credentials. An example of `<albc_role_arn> is arn:aws:iam::<aws_account_number>:role/albo-controller`.
+
[IMPORTANT]
====
Because AWS Load Balancer Controllers do not support creating AWS Load Balancers (ALBs) associated with both AWS Availability Zones and AWS Local Zones, ROSA clusters can have ALBs associated exclusively with either AWS Local Zones or AWS Availability Zones but not both simultaneously.
====
```